src/main/java/stream/flarebot/flarebot/ModlogEvents.java

```diff
@@ -81,7 +81,7 @@ else if (event instanceof GenericMessageEvent && ((GenericMessageEvent) event).g
         if (g == null)
             return;
 
-        GuildWrapper guildWrapper = FlareBotManager.instance().getGuildNoCache(g.getId());
+        GuildWrapper guildWrapper = FlareBotManager.instance().getGuild(g.getId());
```

Comment:
We need to do some checking of how many times this is called, see if maybe we can reduce the number before caching the guild.

Reply:
We really need a better system for tracking things like this :P